smithy-syntax/src/main/java/software/amazon/smithy/syntax/TreeType.java

```diff
@@ -486,8 +469,7 @@ void parse(CapturingTokenizer tokenizer) {
         @Override
         void parse(CapturingTokenizer tokenizer) {
             tokenizer.withState(this, () -> {
-                tokenizer.expect(IdlToken.IDENTIFIER);
-                tokenizer.next(); // skip "input"
+                IDENTIFIER.parse(tokenizer); // skip "input"
```

Comment:
This is `%s"input"` in the grammar, which would be a TOKEN, right? See also output and errors below this.